### PR TITLE
Update troubleshooting guide

### PR DESCRIPTION
**Pull request by amagnus**
Include step to run `rtfctl status` to get an accurate assessment of cluster health.

### CODE REVIEW

**Comment by paigehargrave** [comment]
@amagnus - Thank you for submitting this! Sundip and I are actually working on additional information for this exact thing. Hopefully will have more very soon.

**Comment by amagnus** [comment]
@paigehargrave Any update on this?

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 1]
```diff
@@ -6,6 +6,47 @@ endif::[]
 This topic describes common errors and steps to resolve them when installing Anypoint Runtime
 Fabric.
 
+== Troubleshoot Network Connectivity Using rtfctl
+
+Every Anypoint Runtime Fabric cluster requires connectivity with Anypoint control plane, and any interference with connectivity can limit functionality, resulting in application deployment failures or degraded status in Anypoint Runtime Manager.
+
+=== Obtain a Full Assessment
```
Comment (sundippatel) [inline modules/ROOT/pages/troubleshoot-guide.adoc]: This section is more of a first-level diagnosis of the health of Runtime Fabric. It should be the first thing our customers do when they suspect an issue.

What I think would make sense is if we move this to its own section above the "Troubleshoot Network Connectivity" section.

**Review by sundippatel — CHANGES_REQUESTED** [review]
Suggested to move the instructions involving rtfctl status to be the top-level step for basic troubleshooting. The rest looks good to me.

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 2]
```diff
@@ -6,6 +6,47 @@ endif::[]
...
+https://ecr.us-east-1.amazonaws.com ✔
+https://494141260463.dkr.ecr.us-east-1.amazonaws.com ✔
+https://prod-us-east-1-starport-layer-bucket.s3.amazonaws.com ✔
+https://runtime-fabric.s3.amazonaws.com ✔
+tcp://dias-ingestor-nginx.prod.cloudhub.io:5044 ✔
+```
+ 
+If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have the required whitelisting completed as described in xref:runtime-fabric::install-port-reqs#port-ips-and-hostnames-to-whitelist.
+ 
+For assistance using a TCP dump, refer to: “How to Capture Network Traffic Between Two Systems” (need a link to this KB article).
```
Comment (sundippatel) [inline modules/ROOT/pages/troubleshoot-guide.adoc]: Let's remove this line. We may decide to bring in the content from the support KB at a later time.

**Review by paigehargrave — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-guide.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/troubleshoot-guide.adoc]: Got it - thank you!

**Review by sundippatel — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-guide.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/troubleshoot-guide.adoc]: Got it - thanks!

**Pull request merged**